includes/polling/os/ekinops.inc.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
This file should be converted to the new-ish format, check here for more info, https://docs.librenms.org/Developing/os/Initial-Detection/#os-discovery